src/rprocessing/RLangPApplet.java

```diff
@@ -214,8 +214,9 @@ protected void wrapProcessingVariables() {
         this.renjinEngine.put("displayHeight", displayHeight);
         this.renjinEngine.put("focused", focused);
         this.renjinEngine.put("keyPressed", keyPressed);
-        this.renjinEngine.put("frameCount", frameCount);
+        this.renjinEngine.put("frameCount", (double) frameCount);
```

Comment:
This cast-to-double issue occurs frequently throughout the code -- Processing Java developers are used to using floats for everything, so it is worth adding a note in the Wiki about why it is necessary.